### PR TITLE
feat: add back quote pattern

### PR DESCRIPTION
**Pull request by pirobeem**
I try to add pattern (`) for IE.
Please check it.

Thank you.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "escape-html",
   "description": "Escape string for use in HTML",
-  "version": "1.0.3",
+  "version": "1.0.4",
```
Comment (dougwilson) [inline package.json]: Please remove version bump. This is done on publish.

Comment (pirobeem) [reply package.json]: oh, got it. Thank you!.
I have done rollback version.

**Comment by pirobeem** [comment]
> Thanks so much! Can you add a link to a source where this is needed, update the readme to note this is now escaped too.

source links are following.
[https://it-geek.xyz/post_535](https://it-geek.xyz/post_535)
[http://d.hatena.ne.jp/hasegawayosuke/20070411/p1](http://d.hatena.ne.jp/hasegawayosuke/20070411/p1)

Back quote("`") is special character on IE. So I think it's better to escape it.

**Comment by dougwilson** [comment]
Thanks! Unfortunately I cannot read Japanese and the translate is not really helping. Is there somewhere that explains in english? Or perhaps you can explain here in your own word, perhaps even include a demonstration of how not having this causes an issue and this this will fix said issue? Thanks!

**Comment by pirobeem** [comment]
Oh, sorry for showing Japanese sites.

I tried to check back quote perform on my IE11.
But issue doesn't occur. IE11 escape back quote to "%60".

Maybe issue occur on IE old version. 
but now I can not show that the problem will occur, so I declined this Pull Request.

Sorry for bothering you and thank you for your support :)

**Pull request closed without merging**